.github/workflows/linux-qe-test.yml

```diff
@@ -0,0 +1,29 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Windows artifacts]
+    types:
+      - completed
+  
+jobs:
+  linux-e2e-ocp:
```

Comment:
why linux-e2e-ocp? may better linux-qe

as it includes integration, e2e and microshift no?